### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by mdurrani808**
https://en.wiktionary.org/wiki/%CF%87%CE%BF%CF%81%CE%B5%CF%8D%CF%89
https://en.bab.la/dictionary/english-greek/dancing

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,3 +53,8 @@ If none of these techniques are possible, a reasonable estimate of MoI would be
 ## Tauri Recommended IDE Setup
 
 - [VS Code](https://code.visualstudio.com/) + [Tauri](https://marketplace.visualstudio.com/items?itemName=tauri-apps.tauri-vscode) + [rust-analyzer](https://marketplace.visualstudio.com/items?itemName=rust-lang.rust-analyzer)
+
+## Pronounciation
+
+### Khoreúō /  khor - eh - vo
```
Comment (calcmogul) [inline README.md:59]: That's not the correct pronunciation.

Comment (jlbabilino) [reply README.md:59]: It’s “choreo” as in “choreography”.

**Comment by jlbabilino** [comment]
#77

**Comment by calcmogul** [comment]
OBE by #77.

**Pull request closed without merging**